Documentation/CRDs/ceph-client-crd.md

```diff
@@ -34,10 +38,121 @@ metadata:
   namespace: rook-ceph
 spec:
   caps:
-    mon: 'profile rbd'
+    mon: 'profile rbd, allow r'
     osd: 'profile rbd pool=volumes, profile rbd pool=vms, profile rbd-read-only pool=images'
 ```
 
-### Prerequisites
+To use `CephClient` to connect to a Ceph cluster:
 
-This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+### 2. Find the generated secret for the `CephClient`
+
+Once your `CephClient` has been processed by Rook, it will be updated to include your secret:
+
+```console
+kubectl -n rook-ceph get cephclient example -o jsonpath='{.status.info.secretName}'
+```
+
+### 3. Extract Ceph cluster credentials from the generated secret
+
+Extract Ceph cluster credentials from the generated secret (note that the subkey will be your original client name):
+
+```console
+kubectl --namespace rook-ceph get secret rook-ceph-client-example -o jsonpath="{.data.example}" | base64 -d
```

Comment:
What would the secret name be for this specific example?

Reply:
it would be `rook-ceph-client-sqlitevfs`, will change it to that!

- [ ] change `rook-ceph-client-example` to `rook-ceph-client-sqlitevfs`